docker-bake.hcl

```diff
@@ -219,6 +219,7 @@ target "contracts-bedrock" {
   dockerfile = "./ops/docker/Dockerfile.packages"
   context = "."
   target = "contracts-bedrock"
-  platforms = split(",", PLATFORMS)
+  # See comment in Dockerfile.packages for why we only build for linux/amd64.
+  platforms = "linux/amd64"
```

Comment:
Looks like this is a syntax error, `platforms` might be defined as a list so perhaps `["linux/amd64"]`

Reply:
Shoot you're right - fixed.